scripts/import_dataset.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -e
+# Copyright 2020. IBM All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BASEDIR="$(dirname "$0")"
+source ${BASEDIR}/env.sh
+
+# Install aria and get example dataset zip file
```

Comment:
This is such a small file that installing aria, downloading it, and uninstalling aria takes longer than just wget'ing it. So delete all the aria stuff from line 19 to 23

Reply:
Ah okay, thanks! Will update to use wget and output the file as a set name to eliminate the need for that DATASET_NAME variable as well.